scala/ppml/src/test/scala/com/intel/analytics/bigdl/ppml/vfl/PSISpec.scala

```diff
@@ -45,4 +47,21 @@ class PSISpec extends FlatSpec with Matchers with BeforeAndAfter{
     val intersection = pSI1.downloadIntersection()
     require(intersection.size() == 1, "Intersection number is wrong.")
   }
+  "PSI download null intersection" should "work" in {
```

Comment:
Changed. The behavior could be seen in this UT

when there is no other clients, get null intersection, would return a `TimeoutException`

when the client itself does not upload set, it would throw `Exception`